src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.dynamicsecurityanalysis.server.dto;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public enum DynamicSecurityAnalysisStatus {
+    NOT_DONE,
+    RUNNING,
+    CONVERGED,
```

Comment:
To be checked. Could be : 
COMPLETED,
FAILED

Reply:
This is the status of server which includes status of computing result, in every computing server actually has 4 such values

Reply:
No, we have these statuses for loadflow-server (which is fully righteous), for security-analysis-server (which is not really relevant) and for dynamic-simulation-server (I don't know if it's relevant). We have different statuses for shortcircuit, sensi, voltage-init.

Reply:
The notion of convergence is something specific

Reply:
SUCCEED and FAILED are to status used in the front-end, at moment dynamic security analysis has no specific status

Is OK with SUCCEED and FAILED @etiennehomer

Reply:
DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c